modify minitest doc. Add description for minitest-ci.gem (#184)
launchableinc/cli: docs/test-runners/minitest.md

## Recording test results

> Use [minitest-ci](https://github.com/circleci/minitest-ci) to output the test results to a file in minitest. If you already have a CI server storing your test results it may already be installed.

Reviewer: ```suggestion
First, use [minitest-ci](https://github.com/circleci/minitest-ci) to output the test results to a file. If you already store your test results on your CI server, it may already be installed.
```
